src/phases/tpch/normalized/Q2.yml

```diff
@@ -1,82 +1,91 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/product-query"
 Description: |
-  Run TPC-H query 2 (see http://tpc.org/tpc_documents_current_versions/pdf/tpc-h_v3.0.0.pdf) against the normalized schema. Using an 'executionStats' explain causes each command to run its execution plan until no
-  documents remain, which ensures that the query executes in its entirety.
+  Run TPC-H query 2 (see http://tpc.org/tpc_documents_current_versions/pdf/tpc-h_v3.0.0.pdf) against the normalized schema.
```

Comment:
What happened with this description change? It still runs explain with 'executionStats'?

Reply:
I moved that comment to the workload definition in commit 2.